### PR TITLE
Use CLI in quickstart

### LINKED ISSUE #246: Use CLI in quick start

### PR DESCRIPTION
**Pull request by gvdongen**
Fixes #246 
- Add option to use the CLI for the quickstart

### CODE REVIEW

**Comment by gvdongen** [comment]
Test build is failing due to broken link to `/restate/cli#installation`
--> Will be solved once PR #249 is merged

File: docs/quickstart.mdx [hunk 1]
```diff
@@ -104,54 +110,69 @@ Run the Restate Docker container:
...
 are running behind this deployment and will keep track of these (including of the method signatures).
 After executing this curl request, you should see the discovered services and methods printed to your terminal.
 
+</TabItem>
+<TabItem value="mac" label="macOS">
+
+Run the Restate Docker container:
+
+```shell
+docker run --name restate_dev --rm -d -p 8080:8080 -p 9070:9070 -p 9071:9071 docker.io/restatedev/restate:VAR::RESTATE_VERSION
```
Comment (AhmedSoliman) [inline docs/quickstart.mdx:146]: I think you can drop the docker.io part, no?

Comment (gvdongen) [reply docs/quickstart.mdx:146]: I think this didn't work for some setups. I am not sure anymore which one though. Let me check...

**Review by AhmedSoliman — APPROVED** [review]

File: docs/quickstart.mdx (hunk 1 above)
Comment (jackkleeman) [reply docs/quickstart.mdx:146]: agreed re dropping it
we need 9072 here - see https://github.com/restatedev/documentation/pull/254

Comment (gvdongen) [reply docs/quickstart.mdx:146]: It's required for podman apparently

**Pull request merged**